packages/web-opds-client/tsconfig.json

```diff
@@ -14,6 +14,7 @@
   },
   "exclude": ["node_modules", "lib", "dist"],
   "typedocOptions": {
+    "entryPointStrategy": "expand",
```

Comment:
Explanation: "expand" behavior was the default before v0.22. For v0.22+, "resolve" behavior is the default. We're updating from v0.20... to v0.23..., so this change is to retain the "expand" behavior.

From [the docs](https://typedoc.org/api/enums/EntryPointStrategy.html):
> The default behavior in v0.21 and earlier. Behaves like the resolve behavior, but will recursively expand directories into an entry point for each file within the directory.